### DISCUSSION TITLE
关于 CurseForge API Key

### DISCUSSION THREAD

**Discussion by sulingjiang** [Meloong-Git/PCL#4302] [提问] [answered]
因为没有用过 CurseForge，不是很懂相关政策，所以有几个问题想问问大家

## API Key 有没有有效期，还是说申请一次 CurseForge 倒之前可以一直用？

**Comment by allMagicNB** [comment]
你自己去申请一个……

**Reply by sulingjiang** [reply]
主要是，不清楚作用的话，申请错了还要重新申请，我不是很喜欢麻烦别人，所以非必要的话，我一般不会要求别人重复做同一件事，在我看来这是一种浪费别人时间和精力，对别人极度不尊重的行为

**Comment by hejiehao** [comment, marked as answer]
它应该有一个 [CurseForge for Studios](https://console.curseforge.com/?#/login)，还有一个 [CurseForge 3rd Party API](https://forms.monday.com/forms/dce5ccb7afda9a1c21dab1a1aa1d84eb?r=use1) ，详情看[这个](https://docs.curseforge.com/#what-is-curseforge-for-studios)和[这个](https://support.curseforge.com/en/support/solutions/articles/9000208346-about-the-curseforge-api-and-how-to-apply-for-a-key)
至于限制……我只能找到 https://support.curseforge.com/en/support/solutions/articles/9000207405-curse-forge-3rd-party-api-terms-and-conditions

**Reply by sulingjiang** [reply]
行，我去看看，谢谢
